.github/config.yml

```diff
@@ -12,9 +12,24 @@ newIssueWelcomeComment: |
 newPRWelcomeComment: |
   💖 Thanks for opening this pull request! 💖
 
-  ![typing cat](https://user-images.githubusercontent.com/2289/36400158-2c7c589e-1584-11e8-81c7-bd34fd3c392b.gif)
+  We use [semantic commit messages](https://conventionalcommits.org/) to streamline the Electron release process. Before 
+  your pull request can be merged, you must **update your pull request title** 
```

Comment:
Instead of saying "you must update your pull request title", how about "please ensure that your pull request title...."